Packs/Indeni/Integrations/Indeni/Indeni.py

```diff
@@ -0,0 +1,360 @@
+        if block.get('body') is not None:
+            details = details + block.get('body') + '\n'
+    incident['details'] = details
+
+    return incident
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def test_module(base_url):
+    """
+    Performs basic get request to get item samples
+    """
+    http_request('GET', base_url + 'issues?page=1&per_page=2&sort_by=updated_at.desc')
+
+
+def get_device_request(device_id, base_url):
+    # The service endpoint to request from
+    endpoint_url = 'devices/' + device_id
+    # Send a request using our http_request wrapper
+    response = http_request('GET', base_url + endpoint_url)
+    return response
+
+
+def get_alert_detail_request(alert_id, base_url):
+    endpoint_url = 'issues/' + alert_id
+    response = http_request('GET', base_url + endpoint_url)
+    return response
+
+
+def get_all_active_issues(per_page, sort_by, base_url):
+    issues = []
+    # The service endpoint to request from
+    endpoint_url = 'issues'
+    # Dictionary of params for the request
+    params = {
+        'page': 1,
+        'per_page': per_page,
+        'sort_by': sort_by,
+        'resolved': 'false'
+    }
+    # Send a request using our http_request wrapper
+    response = http_request('GET', base_url + endpoint_url, params)
+    # testing remove the loop
+    while response:
+        issues.extend(response)
+        params['page'] = params['page'] + 1
+        response = http_request('GET', base_url + endpoint_url, params)
+    return issues
+
+
+def get_limited_active_issues(per_page, alert_id_index, size, only_Pan_Cve_Issues, issue_severities, base_url):
+    issues = []
+    issue_count = 0
+    # The service endpoint to request from
+    endpoint_url = 'issues'
+    # Dictionary of params for the request
+    params = {
+        'page': 1,
+        'per_page': per_page,
+        'sort_by': 'alert_id.asc',
+        'resolved': 'false'
+    }
+    # Send a request using our http_request wrapper
+    response = http_request('GET', base_url + endpoint_url, params)
+    # testing remove the loop
+    while response:
+        for item in response:
+            alert_id = item.get('alert_id')
+            if alert_id <= alert_id_index:
+                continue
+            else:
+                alert_id_index = alert_id
+                if (only_Pan_Cve_Issues is False) or \
+                        (only_Pan_Cve_Issues and item.get('unique_identifier').startswith('panos_vulnerability')):
+                    alert_severity = item.get('severity').get('description')
+                    if alert_severity in issue_severities:
+                        issues.append(item)
+                        issue_count = issue_count + 1
+                        if issue_count == size:
+                            return issues, alert_id_index
+        params['page'] = params['page'] + 1
+        response = http_request('GET', base_url + endpoint_url, params)
+    return issues, alert_id_index
```

Comment:
please test this with more than 20 alerts and make sure you get all the alerts after 2 fetches and do not get duplicates.

Reply:
Added another unit test to test the size parameter